src/app/components/Form/Form.tsx

```diff
@@ -32,44 +35,66 @@ export default function Form({
   }
 
   return (
-    <FormContainer onSubmit={handleSubmit}>
-      <label htmlFor="name">{nameLabel}:</label>
-      <Input
-        type="text"
-        id="name"
-        placeholder="Lena Beispiel"
-        onChange={(event) => setName(event.target.value)}
-        value={name}
-        submitted={inputError}
-      />
-      {inputError && name === '' && (
-        <SubmitWarning>Bitte geben Sie einen Namen ein.</SubmitWarning>
-      )}
-      <label htmlFor="evaluation">{evaluationLabel}:</label>
-      <Textarea
-        id="evaluation"
-        rows={3}
-        placeholder="Lena arbeitet häufig gut mit."
-        onChange={(event) => setEvaluation(event.target.value)}
-        value={evaluation}
-        submitted={inputError}
-      />
-      {inputError && evaluation === '' && (
-        <SubmitWarning>Bitte geben Sie eine Beurteilung ein.</SubmitWarning>
-      )}
-      <SubmitButton type="submit" value="Hinzufügen" />
-    </FormContainer>
+    <FormWrapper>
+      <FormContainer onSubmit={handleSubmit}>
+        <RoundActionButton
+          children="X"
+          handleClick={() => onCancel()}
+          customStyles={{
+            sizeButton: '1.5rem',
+            topPosition: '0.375rem',
+            rightPosition: '0.5rem',
+          }}
+        />
+        <label htmlFor="name">{nameLabel}:</label>
+        <Input
+          type="text"
+          id="name"
+          placeholder="Lena Beispiel"
+          onChange={(event) => setName(event.target.value)}
+          value={name}
+          submitted={inputError}
```

Comment:
What does this do exactly?

Reply:
It checks whether an input is missing after submitting. I renamed it to "missingInput", so it should be more descriptive.